sbd.spec

```diff
@@ -97,6 +97,7 @@ Available rpmbuild rebuild options:
 Summary:        Storage-based death environment for regression tests
 License:        GPL-2.0-or-later
 Group:          System Environment/Daemons
+Requires:       libaio-devel
```

Comment:
The reason for the requirement here is not obvious. So we might want to have a comment to first remember why we did it and to stumble over it to find a better solution (best before the next release).